tutorials/capabilities/capabilities.md

```diff
@@ -246,6 +247,14 @@ The third line stating the number of slots in the CSpace, is incorrect, and your
 
 /*-- endfilter -*/
 
+<details markdown='1'>
+<summary style="display:list-item"><em>Quick solution</em></summary>
+
+```c
+    size_t initial_cnode_object_size_bytes = initial_cnode_object_size * (1u << seL4_SlotBits);
+```
```

Comment:
So, while it is very cool to have the solutions displayed, including the code manually duplicates it from the code solution and what is displayed on the page is not actually tested anywhere. That means it will inevitably break at some point, which will be hard to notice and fix. We should investigate some way to extract this from a single source that is actually tested.

Reply:
Fair point. Instead of duplicating the code in the solutions, I'll use the solutions that are already in the md files and are created by running
```
./init --solution --tut capabilities
```
I can test the new md files in the tutorials-manifest.